src/access/universal_v1.test.ts

```diff
@@ -889,16 +891,21 @@ describe("setPublicAccess", () => {
       acpLowLevel,
       "getResourceInfoWithAcr"
     );
+
     const mockedResource = mockSolidDatasetFrom(
       "https://arbitrary.pod/resource"
     );
+
     const mockedAcr = mockAcrFor("https://arbitrary.pod/resource");
     const mockedResourceWithAcr = addMockAcrTo(mockedResource, mockedAcr);
     getResourceInfoWithAcr.mockResolvedValueOnce(mockedResourceWithAcr);
     jest.spyOn(acpModule, "internal_setPublicAccess");
 
-    await setPublicAccess("https://arbitrary.pod/resource", { read: true });
-
+    await setPublicAccess(
+      "https://arbitrary.pod/resource",
+      { read: true },
+      { fetch: jest.fn<typeof fetch>() }
+    );
```

Comment:
This has been added because of what I think are weird `jsdom` issues in 3420add2238afd308e38d52f28bf64e32dbdcbcf, and these specific tests are actually very shallow, they just check that the appropriate module is called. The actual tests are in each module, so I wouldn't be worried here about backwards compatibility.